web/src/api/axios.ts

```diff
@@ -0,0 +1,33 @@
+import axios, { AxiosInstance } from 'axios';
+
+const instance: AxiosInstance = axios.create({
+  baseURL: 'https://capstone-mock-api.fly.dev',
+  // timeout: 15000,
+});
+
+// 요청 인터셉터 추가하기
+instance.interceptors.request.use(
+  (config) => {
+    // 요청이 전달되기 전에 작업 수행
+    return config;
+  },
+  (error) => {
+    // 요청 오류가 있는 작업 수행
+    return Promise.reject(error);
+  }
+);
+
+// 응답 인터셉터 추가하기
+instance.interceptors.response.use(
```

Comment:
Same above. If you've tried this, is there any priority for orders like `try catch` and `interceptors.response`?

Reply:
Q. Does the option works together? No? Then which one?

Reply:
`interceptor` works first. if error occured, `interceptor` return `Promise.reject(error)` then `catch` can catch error.
But if `interceptor` return `Promise<any>` then `catch` can't catch error.

So we can use `interceptor` for repeatitive error handle, and also `catch` if needed.

Reply:
Very nice! So we can use both `try catch` and `interceptor` together. Thank you!